### ISSUE TITLE
Cannot publish package to npm

### ISSUE THREAD

**Issue by Sleavely** [Sleavely/json-changes#1]
```
npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/diffjson - Package name too similar to existing package diff-json
```

**Comment by Sleavely** [comment]
I've reached out to the author of `jsondiff` to see if they would be willing to transfer ownership of the name since it's been in v0.0.0 for 9 years.

**Comment by Sleavely** [comment]
Fixed in ca7cd0e9ad977ec059258e5d6d83e01155a619b2
